addon/components/frost-table.js

```diff
@@ -60,6 +61,24 @@ export default Component.extend({
 
   // == Lifecycle Hooks =======================================================
 
+  didInsertElement () {
+    run.schedule('afterRender', this, function () {
+      const that = this
```

Comment:
ES6 has a concept of "fat arrow functions" that allows you to retain external context inside a function 
https://developer.mozilla.org/en/docs/Web/JavaScript/Guide/Functions#Arrow_functions

Please use these instead of retaining the external context in a local variable